CHANGELOG.md

```diff
@@ -4,6 +4,78 @@ All notable changes to this project will be documented in this file.
 
 ## Unreleased
 
+## Unreleased (2021-01-15)
+
+### :boom: Breaking Change
```

Comment:
Maybe all breaking changes should get some hints in the upgrade guidelines section of the readme?